lib/signs/utilities/messages.ex

```diff
@@ -50,28 +50,32 @@ defmodule Signs.Utilities.Messages do
           Engine.Alerts.Fetcher.stop_status()
         ) :: sign_messages()
   defp get_headway_or_alert_messages(sign, current_time, alert_status) do
-    case get_alert_messages(alert_status) do
+    case get_alert_messages(alert_status, sign.uses_shuttles) do
       nil -> Signs.Utilities.Headways.get_messages(sign, current_time)
       messages -> messages
     end
   end
 
-  @spec get_alert_messages(Engine.Alerts.Fetcher.stop_status()) :: sign_messages() | nil
-  defp get_alert_messages(alert_status) do
-    case alert_status do
-      :shuttles_transfer_station ->
+  @spec get_alert_messages(Engine.Alerts.Fetcher.stop_status(), boolean()) ::
+          sign_messages() | nil
+  defp get_alert_messages(alert_status, uses_shuttles) do
+    case {alert_status, uses_shuttles} do
+      {:shuttles_transfer_station, _} ->
```

Comment:
`:shuttles_transfer_station` is a new case you've added here.  What does that mean?  How is it different from `:suspension_transfer_station`?

Reply:
that one already existed! I think it just looks like an entirely new case with how Git is diffing the versions. But to clarify the different `alert_status`es, `:shuttle_transfer_station` is basically when there's no train service running but there are shuttles for riders to transfer to and `:suspension_transfer_station` would mean there is no service but also no shuttles. I'm not exactly sure in what real world situations the suspension alert is used though.

So the